From 26509a4e48000c62e8b374cb9a7c93220bda4aa0 Mon Sep 17 00:00:00 2001
From: Khai Wen Ng <khai.wen.ng@intel.com>
Date: Thu, 11 Sep 2025 04:09:37 +0800
Subject: [PATCH 36/61] media: i2c: add ACPI_ID for isx031

Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/i2c/isx031.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/media/i2c/isx031.c b/drivers/media/i2c/isx031.c
index ddc3e512efa49..872d466a2c655 100644
--- a/drivers/media/i2c/isx031.c
+++ b/drivers/media/i2c/isx031.c
@@ -813,6 +813,12 @@ static int isx031_probe(struct i2c_client *client)
 	return ret;
 }
 
+static const struct acpi_device_id isx031_acpi_ids[] = {
+        { "INTC1031" },
+        {}
+};
+MODULE_DEVICE_TABLE(acpi, isx031_acpi_ids);
+
 static const struct dev_pm_ops isx031_pm_ops = {
 	SET_SYSTEM_SLEEP_PM_OPS(isx031_suspend, isx031_resume)
 };
@@ -827,6 +833,7 @@ static struct i2c_driver isx031_i2c_driver = {
 	.driver = {
 		.name = "isx031",
 		.pm = &isx031_pm_ops,
+		.acpi_match_table = ACPI_PTR(isx031_acpi_ids),
 	},
 	.probe = isx031_probe,
 	.remove = isx031_remove,
-- 
2.50.1

