From 2802d5f1086d6d14c55a0164b78ac9adb49fed4b Mon Sep 17 00:00:00 2001
From: Khai Wen Ng <khai.wen.ng@intel.com>
Date: Thu, 11 Sep 2025 04:10:27 +0800
Subject: [PATCH 37/61] media: i2c: add ACPI ID for max9296a

Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/i2c/maxim-serdes/max9296a.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/media/i2c/maxim-serdes/max9296a.c b/drivers/media/i2c/maxim-serdes/max9296a.c
index a67692b2f5ee0..2efb70e89acb4 100644
--- a/drivers/media/i2c/maxim-serdes/max9296a.c
+++ b/drivers/media/i2c/maxim-serdes/max9296a.c
@@ -1317,6 +1317,12 @@ static const struct max9296a_chip_info max96792a_info = {
 	.phy_hw_ids = { 1, 2 },
 };
 
+static const struct acpi_device_id max9296a_acpi_ids[] = {
+        { "INTC1137", (kernel_ulong_t)&max9296a_info },
+        {}
+};
+MODULE_DEVICE_TABLE(acpi, max9296a_acpi_ids);
+
 static const struct of_device_id max9296a_of_table[] = {
 	{ .compatible = "maxim,max9296a", .data = &max9296a_info },
 	{ .compatible = "maxim,max96714", .data = &max96714_info },
@@ -1332,6 +1338,7 @@ static struct i2c_driver max9296a_i2c_driver = {
 	.driver	= {
 		.name = "max9296a",
 		.of_match_table	= max9296a_of_table,
+		.acpi_match_table = ACPI_PTR(max9296a_acpi_ids),
 	},
 	.probe = max9296a_probe,
 	.remove = max9296a_remove,
-- 
2.50.1

