From 7cfc9ad5c85b279a7106c71de9613fe495f8e4c7 Mon Sep 17 00:00:00 2001
From: Khai Wen Ng <khai.wen.ng@intel.com>
Date: Thu, 11 Sep 2025 05:09:19 +0800
Subject: [PATCH 41/61] media: i2c: isx031: initialize v4l2 subdev state

Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/i2c/isx031.c | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/drivers/media/i2c/isx031.c b/drivers/media/i2c/isx031.c
index 603db0f8d83a4..05d5f818c19bb 100644
--- a/drivers/media/i2c/isx031.c
+++ b/drivers/media/i2c/isx031.c
@@ -836,7 +836,19 @@ static int isx031_probe(struct i2c_client *client)
 		goto probe_error_media_entity_cleanup;
 	}
 	isx031->cur_mode = isx031->pre_mode;
+
+	ret = v4l2_subdev_init_finalize(&isx031->sd);
+	if (ret < 0) {
+			dev_err(&client->dev, "failed to finalize V4L2 subdev: %d",
+					ret);
+			goto probe_error_media_entity_cleanup;
+	}
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 13, 0)
+	ret = v4l2_async_register_subdev_sensor_common(&isx031->sd);
+#else
 	ret = v4l2_async_register_subdev_sensor(&isx031->sd);
+#endif
 	if (ret < 0) {
 		dev_err(&client->dev, "failed to register V4L2 subdev: %d",
 			ret);
-- 
2.50.1

